pages/collections/[id]/archive.js

```diff
@@ -1,6 +1,7 @@
 import ArchiveList from "@/components/ArchiveList";
 import styled from "styled-components";
 import Link from "next/link";
+import { useRouter } from "next/router";
 
 const Container = styled.div`
```

Comment:
Since the `Container` holds the main page title, section title, and navigation link, it should be a `<header>` instead of a `<div>`. This improves semantic structure, accessibility, and SEO.
```suggestion
const Container = styled.header`
```